flowmachine/flowmachine/core/preflight.py

```
@@ -0,0 +1,99 @@
+        else:
+            # In case we didn't find the attribute and didn't break above.
+            # We should never hit this - it's just here for completeness
+            # to exclude the possibility of attr being undefined.
+            continue
+
+        try:
+            hook_config = attr.__hooks__
+        except AttributeError:
+            pass
+        else:
+            for key in hook_config.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)